### PR TITLE
BUG: Updated changelog instructions

### PR DESCRIPTION
**Pull request by fkneeland-figure**
Closes: #1643 

## Description

______

### PR author checklist:
- [x] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [x] Linked to GitHub issue.
- [x] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [x] Reviewed `Files changed` in the GitHub PR explorer.
- [x] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,22 +88,33 @@ To pull in updates from the origin repo, run
...
 
 ```bash
-unclog add -c ibc improvements 1234-some-issue
+cargo install unclog
 ```
 
-### Add a .changelog entry for the `ibc-relayer-cli` crate (in the `relayer-cli`
-### directory) under the `FEATURES` section in CHANGELOG.md.
+unclog add --editor vim --id allow-fee-granters --section features --issue-no 1633 --message "Added support for fee granters through config file"
+unclog add -i update-unclog-instructions -s bug -n 1634 -m "update CONTRIBUTING.md for latest version of unclog" --editor vim
```
Comment (romac) [inline CONTRIBUTING.md:102]:
```suggestion
    $ unclog add --editor vim --id allow-fee-granters --section features --issue-no 1633 --message "Added support for fee granters through config file"
    $ unclog add -i update-unclog-instructions -s bug -n 1634 -m "Update CONTRIBUTING.md for latest version of unclog" --editor vim
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -88,22 +88,33 @@ To pull in updates from the origin repo, run
...
+
+### Examples
+
+Add a .changelog entry for the `ibc` crate (in the `modules` directory)
+under the `IMPROVEMENTS` section in CHANGELOG.md.
+
+```bash
+unclog add -c ibc -s improvements -i some-issue -n 1234 -n "msg about change" --editor vim
```
Comment (romac) [inline CONTRIBUTING.md:110]:
```suggestion
$ unclog add -c ibc -s improvements -i some-issue -n 1234 -n "msg about change" --editor vim
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -88,22 +88,33 @@ To pull in updates from the origin repo, run
...
 
-### Add a .changelog entry for the `ibc-relayer-cli` crate (in the `relayer-cli`
-### directory) under the `FEATURES` section in CHANGELOG.md.
+unclog add --editor vim --id allow-fee-granters --section features --issue-no 1633 --message "Added support for fee granters through config file"
+unclog add -i update-unclog-instructions -s bug -n 1634 -m "update CONTRIBUTING.md for latest version of unclog" --editor vim
+
+### Examples
+
+Add a .changelog entry for the `ibc` crate (in the `modules` directory)
+under the `IMPROVEMENTS` section in CHANGELOG.md.
```
Comment (romac) [inline CONTRIBUTING.md:107]:
```suggestion
under the `IMPROVEMENTS` section in `CHANGELOG.md`.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -88,22 +88,33 @@ To pull in updates from the origin repo, run
...
+```bash
+unclog add -c ibc -s improvements -i some-issue -n 1234 -n "msg about change" --editor vim
+```
+
+Add a .changelog entry for the `ibc-relayer-cli` crate (in the `relayer-cli`
+directory) under the `FEATURES` section in CHANGELOG.md.
 
 ```bash
-unclog add -c ibc-relayer-cli features 1235-some-other-issue
+unclog add -c ibc-relayer-cli -s features -i some-other-issue -n 1235 -n "msg about change" --editor vim
```
Comment (romac) [inline CONTRIBUTING.md:117]:
```suggestion
$ unclog add -c ibc-relayer-cli -s features -i some-other-issue -n 1235 -n "msg about change" --editor vim
```

**Review by adizere — APPROVED** [review]
Looks good! I did a minor change (https://github.com/informalsystems/ibc-rs/pull/1644/commits/b33ef3d66876e9f48fa25899e3e4ea944827883b) to fix the stale `unclog` examples, and added the new example in the correct section.

**Comment by fkneeland-figure** [comment]
Awesome!  Thanks so much @adizere

**Pull request merged**